client/src/webview-js/image-upload/image-upload.ts

```diff
@@ -45,21 +53,23 @@ window.addEventListener('load', () => {
     }
   }
 
-  const handleDrop = (e) => {
-    const files = e.dataTransfer.files;
+  const handleDrop = (e: DragEvent): void => {
+    const files = e.dataTransfer?.files
 
-    ([...files]).forEach(previewFile)
+    if (files) { // eslint-disable-line @typescript-eslint/strict-boolean-expressions
```

Comment:
I think the linter probably wants
```suggestion
    if (files != null) {
```